src/app/stylesheets/variables.less

```diff
@@ -77,6 +77,8 @@
 @public-marketplace-show-sections-bg-color:           @bg-on-bg-inverse-color;
 @public-marketplace-show-sections-text-color:         @text-strong-color;
 
+@public-marketplace-carousel-card-color:              rgba(255,255,255,.9);
+@public-marketplace-carousel-card-border-color:       #e3e3e3; 
```

Comment:
It would be better to use variables from `theme.less` here instead:
Maybe `@bg-on-bg-inverse-color` for the white and either change one that is unused for the gray or add a new variable for it.